Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rust:1.42.0
+ENV KUBECTL_VERSION=v1.17.3
+ENV CLOUD_SDK_VERSION=283.0.0
+ENV INTERNAL_IP=127.0.0.1
```

Comment:
We should remove the IP related env vars from the image.

Reply:
it was a leftover, removed.